os/drivers/lcd/st7785.c

```diff
@@ -48,6 +49,19 @@ int check_lcd_vendor_send_init_cmd(struct mipi_lcd_dev_s *priv)
 	}
 	lcddbg("LCD ST7785 not recognized\n");
 	return ERROR;
+#elif defined(CONFIG_LCD_STATIC_INIT)
+	/* Use static init code */
+	#ifdef CONFIG_LCD_STATIC_INIT_DEFAULT
+		return send_init_cmd(priv, lcd_init_cmd_g);
+	#elif defined(CONFIG_LCD_STATIC_INIT_AVD)
+		return send_init_cmd(priv, lcd_init_cmd_g_avd);
+	#elif defined(CONFIG_LCD_STATIC_INIT_HOLITECH)
+		return send_init_cmd(priv, lcd_init_cmd_g_holitech);
+	#else
+		lcddbg("No static initialization type selected\n");
+		return ERROR;
+	#endif
+#endif
```

Comment:
```suggestion
#elif defined(CONFIG_LCD_STATIC_INIT)
	/* Use static init code */
#ifdef CONFIG_LCD_STATIC_INIT_DEFAULT
	return send_init_cmd(priv, lcd_init_cmd_g);
#elif defined(CONFIG_LCD_STATIC_INIT_AVD)
	return send_init_cmd(priv, lcd_init_cmd_g_avd);
#elif defined(CONFIG_LCD_STATIC_INIT_HOLITECH)
	return send_init_cmd(priv, lcd_init_cmd_g_holitech);
#else
#error "No static initialization type selected"
#endif
#endif
```

Please check the indentation and the configure is static init so build time error is better.